GraphQLParser.cpp

```diff
@@ -19,9 +19,11 @@ namespace graphql {
 // Given properly-configured yylex, run the parser and return the
 // result.
 static std::unique_ptr<ast::Node> doParse(const char **outError, yyscan_t scanner, bool enableSchema) {
-  Node *outAST;
+  Node *outAST = NULL;
   yy::GraphQLParserImpl parser(enableSchema, &outAST, outError, scanner);
   int failure = parser.parse();
+  if (failure && outAST)
```

Comment:
Please always use braces around if statements. It's too easy to make mistakes otherwise.

I wouldn't normally request changes over this, but while I'm here: `delete nullptr` is safe, no need to test `outAST`.